packages/@azure/servicebus/data-plane/.scripts/prepSamples.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs");
+const path = require("path");
+
+const baseFolder = path.join(__dirname, "../examples/typescript");
+
+processFolders(path.join(baseFolder, "gettingStarted"));
+processFolders(path.join(baseFolder, "advancedFeatures"));
+
+function processFolders(folder) {
+    const files = fs.readdirSync(folder);
+    files.forEach(file => {
+        console.log(`Updating ${file}...`);
+        enableLocalRun(path.join(folder, file))
+    });
+}
+
+function enableLocalRun(file) {
```

Comment:
I would wrap this in a try/catch to make sure that we gracefully handle cases where the file is locked (ie. opened in Notepad or some other process that opens with exclusive access).

Should we have the reverse script as well? I am thinking of the case where you have made changes to the samples on top of these changes (and you don't know about interactive add in git)?

Reply:
I didnt add any error handling on purpose with an intention of the errors to bubble up.
This script is to be run only as part of the inner dev loop. 

If the file is locked or doesnt exist or the `lib` folder doesnt exist at the expected location (as mentioned in the other comment), these are errors that the developer should fix and therefore it made sense to me for the errors to bubble up.

Reply:
The reverse script is a good idea, but we will have to rely on the developer to run them manually. Or did you have something in mind to automate it before the code is checked in?

Reply:
> Should we have the reverse script as well? I am thinking of the case where you have made changes to the samples on top of these changes (and you don't know about interactive add in git)?

What about saving the modified files under some folder that is ignored by git then build and debug from there? Any drawbacks?

Reply:
The changes in this PR are mainly to enable debugging of the samples. If we move the updated files to another folder, then we should remember to debug that file instead of the ones under the `examples` folder. I foresee us forgetting to do so very often :)

Reply:
@ramya-rao-a if we want to get fancy we could use the git hooks but I think that is not worth the effort.

How about a test that runs in CI and checks if you are referencing the right package? If that is too difficult, then I'm fine with this (I simply wanted to automate remembering to undo these changes 😄 ).

Reply:
Done with https://github.com/Azure/azure-sdk-for-js/pull/1259/commits/fd53e647e3ea8b9e262a13c0d4425a814cdb2697